test/cyad/commandline_options.cpp

```diff
@@ -30,7 +30,7 @@ TEST(CommandlineOptions, allOptionsPresent) {
     using Cynara::CmdlineOpts::commandlineOptions;
 
     // A cheap trick to make sure this test is updated, when new options are added
-    ASSERT_EQ(18, commandlineOptions.size());
+    ASSERT_EQ(18u, commandlineOptions.size());
```

Comment:
1) As far as I know, you are still employee of Samsung.
Making such a change - please update Samsung's copyrights.

2) Please make your changes compatible with the way, there were fixed previously, e.g. in commit ed1c9d53f80e2c58b1a524d4e586044424cf7e7e